config/routes.rb

```diff
@@ -7,7 +7,7 @@
     post "approve-2i-review", to: "review#approve_2i_review"
     post :check_links
     post "claim-2i-review", to: "review#claim_2i_review"
-    get "internal-change-notes", to: "interal_change_notes"
+    get "internal-change-notes", to: "internal_change_notes"
```

Comment:
Not sure how tests passed - we have some that call `when_I_visit_the_change_notes_tab` that I would expect to crash the service. One to look at later?

Reply:
I did, and it didn't!

Reply:
You've looked into this and discovered some interesting Rails behaviour! I've amended my commit.